### PR TITLE
README: remove Homebrew-specific things.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Linuxbrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

-----

Make this README more Linuxbrew focused.

Replaces broken https://github.com/Linuxbrew/brew/pull/426.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,12 +95,6 @@ export MANPATH="$(brew --prefix)/share/man:$MANPATH"
...
 
-## Update Bug
-If Homebrew was updated on Aug 10-11th 2016 and `brew update` always says `Already up-to-date.` you need to run:
-```bash
-cd "$(brew --repo)" && git fetch && git reset --hard origin/master && brew update
-```
-
 ## What Packages Are Available?
 1. Type `brew search` for a list.
 2. Or visit [braumeister.org](http://braumeister.org) to browse packages online.
```
Comment (bw-yuhogg) [inline README.md]: We can probably delete this line too.

File: README.md [hunk 2]
```diff
@@ -118,67 +111,19 @@ Second, read the [Troubleshooting Checklist](https://github.com/Linuxbrew/brew/b
...
+If a Linuxbrew formula doesn't work, [open an issue on GitHub](https://github.com/Linuxbrew/homebrew-core/issues) or, even better, submit a pull request.
 
 ## Contributing
-We'd love you to contribute to Linuxbrew or its upstream project, Homebrew. First, please read our [Contribution Guide](https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/Linuxbrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct). Please see our [guidelines](https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md#contributing-to-linuxbrew) on whether to send pull requests to Linuxbrew or Homebrew.
+We'd love you to contribute to Linuxbrew. First, please read our [Contribution Guide](https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/Linuxbrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct). Please see our [guidelines](https://github.com/Linuxbrew/brew/blob/master/CONTRIBUTING.md#contributing-to-linuxbrew) on whether to send pull requests to Linuxbrew or Homebrew.
```
Comment (bw-yuhogg) [inline README.md:117]: Is that `/CODEOFCONDUCT.md@master#code-of-conduct` new Markdown syntax? I've never seen it before.

**Review by bw-yuhogg — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
🚢?

**Comment by bw-yuhogg** [comment]
As with #429, just waiting for @maxim-belkin's approval.

**Review by maxim-belkin — APPROVED** [review]

**Comment by bw-yuhogg** [comment]
Closed in 59a33850248e81f2301c5e534a59a7d660ab3718

Thanks @MikeMcQuaid

**Pull request closed without merging**